.dockerignore

```diff
@@ -1 +1,6 @@
-docker
\ No newline at end of file
+docker/**/*.db*
+docker/**/backups
+docker/deployed/**/prometheus
+docker/deployed/**/grafana/data/**
+!docker/deployed/**/grafana/data/.gitkeep
```

Comment:
The previous rule was too aggressive. We want to do a mimicking of the `.gitignore`, with an extra twist of allowing the `.gitkeep` files.